vite.config.ts

```diff
@@ -119,7 +119,7 @@ export default defineConfig(async env => ({
         }
       }
     }),
-    env.mode === 'production' ? visualizer() : undefined,
```

Comment:
I thought maybe we don't need that plugin in the dev environment?

Reply:
Btw vite's development mode only in server mode. And this plugin only work at build stage